cmd/podmanV2/volumes/create.go

```diff
@@ -0,0 +1,72 @@
+package volumes
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podman/shared/parse"
```

Comment:
Need to copy the code, you cannot pull from cmd/podman.  When we're done that whole directory is going to be rm'ed.